contracts/upgradeable_contracts/OverdrawManagement.sol

```diff
@@ -4,28 +4,38 @@ import "../upgradeability/EternalStorage.sol";
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "./Upgradeable.sol";
 import "./RewardableBridge.sol";
+import "./BasicBridge.sol";
 
-contract OverdrawManagement is EternalStorage, RewardableBridge, Upgradeable {
+contract OverdrawManagement is EternalStorage, RewardableBridge, Upgradeable, BasicBridge {
     using SafeMath for uint256;
 
     event UserRequestForSignature(address recipient, uint256 value);
 
-    function fixAssetsAboveLimits(bytes32 txHash, bool unlockOnForeign) external onlyIfUpgradeabilityOwner {
+    function fixAssetsAboveLimits(bytes32 txHash, bool unlockOnForeign, uint256 valueToUnlock)
```

Comment:
Does it make sense to increase the interface version in the BasicBridge contract?

Reply:
Increased patch version in basic bridge https://github.com/poanetwork/poa-bridge-contracts/pull/247/commits/26a152ee891d3c1b7b83628c4ef38f8a1706ea87

Reply:
Should we change `minor` instead?

Reply:
Right, I wasn't sure about increasing `patch` or `minor`. Increased `minor` here https://github.com/poanetwork/poa-bridge-contracts/pull/247/commits/1e4c35a0d15ae36eebf94df191c5cecaf0703b6a